src/components/breadcrumb/breadcrumb.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from '../links/links';
+import { BreadcrumbText } from '../typography/typography';
+import { theme } from '../layout/theme';
+
+const BreadcrumbContainer = styled.ol`
+  margin: 16px 24px 0;
```

Comment:
I'm not convinced we should keep this "outer spacing" in this component. There are different opinions about this and there is no right or wrong. Personally I feel like it makes everything easier when we stick to the rule of "components don't define any outside spacing". See [this rather long article about component spacing](https://medium.com/fed-or-dead/handling-spacing-in-a-ui-component-library-70f3b22ec89) for an explanation of why this is a difficult topic.

In this particular case I suggest we get rid of at least the horizontal spacing. Personally I'd like to get rid of the margin completely but I'm happy to hear some more opinions on this.